package.json

```diff
@@ -17,7 +17,7 @@
   "license": "MIT",
```

Comment:
I'm not sure why yet, but the update to `@playwright/test` >= 1.20 causes an issue with UMD browser-tests.

Reply:
Problem fixed itself with `@playwright/test` 1.27.